packages/frontend/src/components/Explorer/CustomDimensionModal/index.tsx

```diff
@@ -219,6 +236,160 @@ export const CustomDimensionModal = () => {
                         />
                     )}
 
+                    {form.values.binType === BinType.CUSTOM_RANGE && (
+                        <>
+                            <Text fw={500}>Range</Text>
+                            {form.values.binConfig.customRange.map(
```

Comment:
This `custom range` form can be extracted into a separated component. I'll create a ticket for this to do this on a separate PR